src/action-sheets/index.js

```diff
@@ -236,6 +242,12 @@ const unsubscribe = async ({ auth, streamId, subscriptions }) => {
 unsubscribe.title = 'Unsubscribe';
 unsubscribe.errorMessage = 'Failed to unsubscribe';
 
+const searchMessage = ({ streamId }) => {
+  NavigationService.dispatch(navigateToSearch(streamId));
+};
+searchMessage.title = 'Search Message';
+searchMessage.errorMessage = 'Failed to open search';
```

Comment:
You'll have a rebase conflict to resolve here, sorry -- see f32a679828ca569aabf528fb24002128342b4088 for the relevant change.